src/uniparc/components/entry/Entry.tsx

```diff
@@ -49,7 +49,7 @@ const Entry: FC = () => {
 
   // if URL doesn't finish with "entry" redirect to /entry by default
   useEffect(() => {
-    if (match && !match.params.subPage) {
+    if (match && !match.params.subPage && accession) {
```

Comment:
This should probably be:
```ts
useEffect(() => {
  if (match && !match.params.subPage) {
    history.replace({
      ...history.location,
      pathname: generatePath(LocationToPath[Location.UniParcEntry], {
        accession: match.params.accession,
        subPage: TabLocation.Entry,
      }),
    });
  }
}, [match, history]);
```
In order to be consistent with the logic on the UniProtKB entry page

Reply:
Did you mean to replace the equivalent of `accession: accession` by `accession: match.params.accession` so that that `useEffect` is the exact same across namespaces?

Reply:
Yes, removes the extra check because accession will always be truthy if match is truthy, and makes it more useful that it's the same logic and thus could be abstracted